CODE_OF_CONDUCT.md

```diff
@@ -1,55 +1,38 @@
 # Code of Conduct
```

Comment:
speaking with @tkremenek and @DougGregor they suggested we simply link (make the content of this file into a single link) from all our packages to https://swift.org/code-of-conduct/ which is the canonical CoC for the Swift project. I think this is a good idea, WDYT?

cc @weissi @Lukasa @ktoso @yim-lee

Reply:
Sure, WFM.

Reply:
WFM